_guide/_pages/security.md

```diff
@@ -4,4 +4,81 @@ sidenav: security
 sticky_sidenav: true
 ---
 
-Security is everybody's responsibility at TTS. There are practices that we should adhere to as much as possible when building websites and this guide contains the ones that front-end designers and developers need to be aware of.
+Security is everybody's responsibility at TTS but if you're not used to thinking about security in your day-to-day job, 
+that's understandable. We come from a range of different backgrounds, some of which involved security more than others. 
+
+There are practices that we should adhere to as much as possible when building websites and this guide contains ones 
+that front-end designers and developers need to be aware of. We aim to give everyone a framework to think about security, and
```

Comment:
Just noticing this, but the "designers & developers" line feels a little less relevant as we add more content, I've been thinking of the primary audience for the Engineering Practices Guide as engineers rather than designers